backend/eventloop/shared_resource_loop/sharedresourceloop_managedenv.go

```diff
@@ -572,13 +574,7 @@ func deleteManagedEnvironmentResources(ctx context.Context, managedEnvID string,
 
 		log := log.WithValues("applicationID", app.Application_id)
 
-		// Nil the managed environment field of the Application (since we are deleting the managed environment itself, and we have a foreign key here)
-		app.Managed_environment_id = ""
-		if err := dbQueries.UpdateApplication(ctx, &app); err != nil {
-			log.Error(err, "Unable to update Application row: nil-ing the managed environment field of Application that uses deleted managed environment")
-			return fmt.Errorf("unable to nil managed environment of applicaton '%s': %v", app.Application_id, err)
-		}
-		log.Info("Updated Application row: nil-ing the managed environment field of Application that uses deleted managed environment")
+		log.Info("Updated Application row: the managed environment field was nil-ed, of Application that uses deleted managed environment")
```

Comment:
Should we remove this log since the Application rows are already nil-ed before the loop?